src/modules/Routes/BookList.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Book from './Book';
+import NewBook from './NewBook';
+
+function BookList() {
+  return (
+    <div>
+      <Book />
+      <Book />
+      <Book />
+      <NewBook />
+    </div>
```

Comment:
✅  Renders a list of books using the `Book` component 👍🏻 
✅  Renders a form by using the NewBook component 👍🏻